src/test/java/com/google/javascript/gents/singleTests/functions.js

```diff
@@ -15,6 +15,11 @@ function twoParams(b, s) {}
 /** @param {!Array<?>=} list */
 function withDefaultValue(list = []) {}
 
+/**
+ * @param {boolean=} a
+ */
+function undefinedDefault(a = undefined) {}
```

Comment:
add a test with something a bit more complicated (maybe !Array<number> or even Array<number>) to make sure the union is done right.

Reply:
Done.